lwgenerate/azure/activity_log_with_config.tf

```diff
@@ -0,0 +1,43 @@
+terraform {
```

Comment:
➕1 on this. Currently, AWS and GCP have the test examples inline in the test file. However, it may make sense to have these stored as files in a test resources dir.

Reply:
+1

Reply:
Created a test-data folder under the azure folder to hold all these.